docs/index.rst

```diff
@@ -46,12 +52,12 @@ License: BSD-3
 News
```

Comment:
A lot of projects (e.g. scikit-learn, scikit-image) have a "News" section on the front page, to mention releases or other major news. I think some easy way for users to see what's new is a good thing.
So I added a "news" section for astroplan and [Gammapy](https://gammapy.readthedocs.org/en/latest/#news), where the Gammapy one points to the changelog.

Really this is only needed starting with the 0.2 release, but I think it might be easiest to set this up right from the start instead of removing it completely. See e.g. [wcsaxes](http://wcsaxes.readthedocs.org/en/latest/) ... it's a great project that has had a few releases, but it's there's no info from the docs what version it's at and what's new.

As other examples, [sncosmo](https://sncosmo.readthedocs.org/en/v1.0.x/) has a "What's new" entry pointing to a changelog. [Astropy](http://astropy.readthedocs.org/en/latest/) has a "What's new" link pointing to a separate what's new page with some code and figures.

@adrn – can we keep some kind of "what's new" info or link on the front page? any suggestion how you'd prefer to have it set it up?